### ISSUE TITLE
Cannot find webpack.build.config.js on installation

### ISSUE THREAD

**Issue by jimutt** [vuejs/vue-resource#186]
I'm trying to install vue-resource through NPM but I'm getting the following error:

```
> vue-resource@0.5.0 build /home/ubuntu/workspace/choralProject/node_modules/vue-resource
> webpack --config build/webpack.build.config.js

module.js:338
    throw err;
    ^

Error: Cannot find module '/home/ubuntu/workspace/choralProject/node_modules/vue-resource/build/webpack.build.config.js'
    at Function.Module._resolveFilename (module.js:336:15)
    at Function.Module._load (module.js:286:25)
    at Module.require (module.js:365:17)
    at require (module.js:384:17)
    at module.exports (/home/ubuntu/workspace/choralProject/node_modules/webpack/bin/convert-argv.js:80:13)
    at Object.<anonymous> (/home/ubuntu/workspace/choralProject/node_modules/webpack/bin/webpack.js:39:40)
    at Module._compile (module.js:434:26)
    at Object.Module._extensions..js (module.js:452:10)
    at Module.load (module.js:355:32)
    at Function.Module._load (module.js:310:12)
```

I'm using the following dependencies and all other dependencies seems to have been installed successfully.
```
  "dependencies": {
    "compression": "^1.0.3",
    "cors": "^2.5.2",
    "loopback": "^2.22.0",
    "loopback-boot": "^2.6.5",
    "loopback-component-explorer": "^2.1.0",
    "loopback-component-storage": "^1.6.0",
    "loopback-connector-mysql": "^2.2.0",
    "loopback-datasource-juggler": "^2.39.0",
    "serve-favicon": "^2.0.1",
    "vue": "^1.0.12",
    "vue-router": "^0.7.7"
  },
  "devDependencies": {
    "babel-core": "^6.3.21",
    "babel-loader": "^6.2.0",
    "babel-plugin-transform-runtime": "^6.3.13",
    "babel-preset-es2015": "^6.3.13",
    "babel-preset-stage-0": "^6.3.13",
    "babel-runtime": "^5.8.34",
    "css-loader": "^0.23.0",
    "eslint": "^1.10.3",
    "eslint-loader": "^1.1.1",
    "file-loader": "^0.8.5",
    "function-bind": "^1.0.2",
    "jade": "^1.11.0",
    "jshint": "^2.5.6",
    "style-loader": "^0.13.0",
    "stylus-loader": "^1.4.2",
    "template-html-loader": "0.0.3",
    "url-loader": "^0.5.7",
    "vue-hot-reload-api": "^1.2.2",
    "vue-html-loader": "^1.0.0",
    "vue-loader": "^7.2.0",
    "webpack": "^1.12.9"
  },
```

**Comment by uxweb** [comment]
@jimutt The same is happening to met too. Have you found out what is the solution?

**Comment by rubennavarroc** [comment]
The same here too.
<quote>
885 error Linux 3.19.0-25-generic
886 error argv "/usr/local/bin/node" "/usr/local/bin/npm" "install" "vue-resource"
887 error node v5.2.0
888 error npm  v3.5.2
889 error code ELIFECYCLE
890 error vue-resource@0.5.0 install: `npm run build && npm run test`
890 error Exit status 1
891 error Failed at the vue-resource@0.5.0 install script 'npm run build && npm run test'.
891 error Make sure you have the latest version of node.js and npm installed.
891 error If you do, this is most likely a problem with the vue-resource package,
891 error not with npm itself.
891 error Tell the author that this fails on your system:
891 error     npm run build && npm run test
891 error You can get information on how to open an issue for this project with:
891 error     npm bugs vue-resource
891 error Or if that isn't available, you can get their info via:
891 error     npm owner ls vue-resource
891 error There is likely additional logging output above.

</quote>

**Comment by xsbear** [comment]
+1

**Comment by NGWNGW** [comment]
+1

**Comment by rubennavarroc** [comment]
I have a workaround for this. It seems that the npm repository for vue-resource is missing the mentioned file. So I downloaded the git repo manually and placed it inside node_modules folder.  After doing so I could use vue-resource normally. Hope it helps.

**Comment by NGWNGW** [comment]
@rubennavarroc Thank you! That works.

**Comment by rubennavarroc** [comment]
@NGWNGW Cool, you're welcome!

**Comment by steffans** [comment]
Fixed in v0.5.1.
